src/home/newmail.ts

```diff
@@ -103,7 +110,7 @@ function createMailForm(): void {
         ],
         height: '100%',
         width: '100%',
-        overflowMode: 'Popup'
+        overflowMode: 'Scrollable'
```

Comment:
WHy mode is changed

Reply:
While re-sizing Splitter Component, Toolbar menus overflow outside the container. To avoid this situation, Overflow mode has been changed.